spec/all_pages_spec.rb

```diff
@@ -19,6 +19,13 @@ def get_locale(path)
 
 RSpec.describe 'all pages' do
   let(:old_paths) { YAML.load_file('OLD_URLS.yml').map { |url| URI(url).path } }
+  let(:redirect_froms) do
+    Dir["#{REPO_ROOT}/content/**/*.md"].map do |path|
+      page = path.split(REPO_ROOT.to_s).last
+      frontmatter = File.read(path).split('---', 3)[1]
+      Array(YAML.load(frontmatter)['redirect_from'])
+    end.flatten
```

Comment:
We could do `flat_map` instead of `map { }.flatten`?

```suggestion
    Dir["#{REPO_ROOT}/content/**/*.md"].flat_map do |path|
      page = path.split(REPO_ROOT.to_s).last
      frontmatter = File.read(path).split('---', 3)[1]
      Array(YAML.load(frontmatter)['redirect_from'])
    end
```